gensim/test/test_word2vec.py

```diff
@@ -19,7 +19,7 @@
 
 from gensim import utils
 from gensim.models import word2vec, keyedvectors
-from gensim.test.utils import datapath, get_tmpfile, common_texts as sentences
+from gensim.test.utils import datapath, get_tmpfile, temporary_file, common_texts as sentences
```

Comment:
Need also check persistence (i.e. successful save/load) + online learning (when you call train twice in row)

Reply:
done